testgrid/cmd/entomologist/README.md

```diff
@@ -0,0 +1,32 @@
+# Entomologist
+
+Entomologist collects bugs from GitHub and pins them to TestGrid tests. When an issue is "pinned"
+to a TestGrid Target, a hyperlink will show up on TestGrid that leads back to the GitHub issue.
```

Comment:
Can note that the hyperlink is a "(B)"

Reply:
A target is a particular test that TestGrid is displaying. This is usually a row.

I've modified the readme to try to explain this implicitly.

Reply:
A target is a particular test that TestGrid is reporting. This is usually a row.

I've changed the readme to try to explain this implicitly